html/changelogs/MenOwOn - Life is Pain.yml

```diff
@@ -0,0 +1,42 @@
+################################
+# Example Changelog File
+#
+# Note: This file, and files beginning with ".", and files that don't end in ".yml" will not be read. If you change this file, you will look really dumb.
+#
+# Your changelog will be merged with a master changelog. (New stuff added only, and only on the date entry for the day it was merged.)
+# When it is, any changes listed below will disappear.
+#
+# Valid Prefixes: 
+#   bugfix
+#   wip (For works in progress)
+#   tweak
+#   soundadd
+#   sounddel
+#   rscadd (general adding of nice things)
+#   rscdel (general deleting of nice things)
+#   imageadd
+#   imagedel
+#   maptweak
+#   spellcheck (typo fixes)
+#   experiment
+#   balance
+#################################
+
+# Your name.  
+author: Menown
+
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes: 
+  - rscadd: "Adds building planting beds with sandstone. Requires Three."
+  - rscadd: "Adds deconstructing the built planting beds. Uses a spade or shovel."
+  - rscadd: "Changes initial spawn of all planting beds, not trays. They all spawn with no water or nutrient."
+  - rscadd: "Removes weed growing from hydroponics trays. Hydroponic trays do not grow weeds. Dumb meme."
```

Comment:
This is a tweak, not rscadd

Reply:
This is a removal, not a tweak. Also a strange removal. How is one of the primary obstacles in hydroponics, weed management, a 'dumb meme'?

Reply:
It is not removing it entirely - it tweaks metalic trays to not have weed, while keeping soil trays weeds. So this is a tweak because some of trays still have it, but some do not

Reply:
Thus making hydroponics the preferred method for sterile farming, instead of soil farming, which has ability for space and capacity, ie: building the plots.

Reply:
Considering soil trays are the trays you have to put effort into creating, I see no reason to remove weeds exclusivities from hydroponic trays and no reason to remove weeds at all.

Reply:
Weeds aren't removed, Fowl. They're only removed from hydroponics trays.